packages/api/.env.example

```
@@ -68,6 +68,18 @@ SIGNED_URL_SECRET=dev_only
 # Expiration applied to signed download URLs.
 SIGNED_URL_EXPIRES_IN=1h
 
+# -----------------------------------------------------------------------------
+# MCP server
+# -----------------------------------------------------------------------------
+# Enables the secured Streamable HTTP MCP endpoint at /api/mcp.
+MCP_ENABLED=true
+# Metadata reported to MCP clients.
+MCP_SERVER_NAME=hexabot-api
+MCP_SERVER_TITLE=Hexabot API MCP Server
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)